plugins/inputs/win_services/README.md

```diff
@@ -8,10 +8,11 @@ Monitoring some services may require running Telegraf with administrator privile
 
 ```toml
 [[inputs.win_services]]
-  ## Names of the services to monitor. Leave empty to monitor all the available services on the host
+  ## Names of the services to monitor. Leave empty to monitor all the available services on the host. Globs accepted.
```

Comment:
I think globs are the technical name that isn't really used outside tech circles. Let's maybe refer to it as wildcards in the docs?

Reply:
I agree, the word "wildcard" would probably be easier to understand. Although the example config that Telegraf produces does use the word "glob" multiple times, do you think we should make an issue to update it as well? 
Example: https://github.com/influxdata/telegraf/blob/7d3b7fc2f9d379aa21d09eb3e905e644e876be6f/etc/telegraf.conf#L2949

Reply:
ok perhaps it's a larger discussion. feel free to ignore

Reply:
ignoring for now then.